tldw_Server_API/tests/Sharing/test_share_audit_unified_migration.py

```
@@ -59,10 +59,51 @@ async def test_share_audit_backfill_is_idempotent(repo, sharing_db, tmp_path):
     assert report1.max_legacy_id == legacy_id
     assert report2.inserted == 0
     assert compatibility_ids == [legacy_id]
+    assert rows[0]["event_id"] == f"share-audit-legacy-{legacy_id}"
     assert rows[0]["created_at"] == legacy_created_at
+    assert rows[0]["metadata"]["legacy_share_audit_id"] == legacy_id
     assert sorted(row["id"] for row in after_rows) == [legacy_id, legacy_id + 1]
 
 
+@pytest.mark.asyncio
+async def test_share_audit_legacy_import_replay_is_skipped(repo, tmp_path):
```

Ruff (0.15.9)

[warning] Unused function argument: `repo`

(ARG001)